src/ai/susi/server/api/cms/ChangeSkillStatusService.java

```diff
@@ -0,0 +1,134 @@
+/**
+ *  ChangeSkillStatusService
+ *  Copyright by @Akshat-Jain on 10/07/18.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program in the file lgpl21.txt
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ai.susi.server.api.cms;
+
+import ai.susi.DAO;
+import ai.susi.json.JsonObjectWithDefault;
+import ai.susi.json.JsonTray;
+import ai.susi.server.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This endpoint allows Admin and higher userroles to change review status of any skill
+ * http://127.0.0.1:4000/cms/changeSkillStatus.json?model=general&group=Knowledge&language=en&skill=aboutsusi&reviewed=true&access_token=zdasIagg71NF9S2Wu060ZxrRdHeFAx
+ */
+
+public class ChangeSkillStatusService extends AbstractAPIHandler implements APIHandler {
+    private static final long serialVersionUID = 7926060917231250102L;
+
+    @Override
+    public UserRole getMinimalUserRole() {
+        return UserRole.ADMIN;
+    }
+
+    @Override
+    public JSONObject getDefaultPermissions(UserRole baseUserRole) {
+        return null;
+    }
+
+    @Override
+    public String getAPIPath() {
+        return "/cms/changeSkillStatus.json";
+    }
+
+    @Override
+    public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Authorization authorization, JsonObjectWithDefault permissions) throws APIException {
+
+        String model_name = call.get("model", "general");
+        String group_name = call.get("group", "All");
+        String language_name = call.get("language", "en");
+        String skill_name = call.get("skill", null);
+        String reviewed = call.get("reviewed", null);
+
+        if (authorization.getIdentity() == null) {
+            throw new APIException(422, "Bad access token.");
+        }
+
+        if (skill_name == null || !(reviewed.equals("true") || reviewed.equals("false"))) {
```

Comment:
@Akshat-Jain I think it should be `(!reviewed.equals("false"))` since we want to accept both the values `true` and `false`?

Reply:
@hkedia321 The `!` is outside the main bracket. Please check it once more. I have tested this locally. I think it's correct. Thanks :-)